pkg/middleware/middleware_test.go

```diff
@@ -43,7 +45,7 @@ func middlewareFilterApp(filterFile, olympusEndpoint string) *buffalo.App {
 }
 
 func newTestFilter(filterFile string) *module.Filter {
-	f := module.NewFilter(filterFile)
+	f, _ := module.NewFilter(filterFile)
```

Comment:
@manugupt1 do you need to check the error here?

Reply:
👍 even if this is test it should be checked for not nil err

Reply:
yep!